### PR TITLE
add save earth hackbar

### PR DESCRIPTION
**Pull request by vikranthdatta**

### CODE REVIEW

File: _hackbars/vikranth.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: climate change
+author: vikranth
+---
+
+open source is one of the most efficient ways in which one can contribute. i personally feel it to be a great way to contribute together with different people having different ideas.
```
Comment (tasnimzotder) [inline _hackbars/vikranth.md]: hey @vikranthdatta Can you please update the line -

```suggestion
Open source is one of the most efficient ways in which one can contribute. I personally feel it to be a great way to contribute together with different people having different ideas.
```

Comment (vikranthdatta) [reply _hackbars/vikranth.md]: ok

Comment (vikranthdatta) [reply _hackbars/vikranth.md]: check it now.

**Review by tasnimzotder — DISMISSED** [review]

File: _hackbars/vikranth.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+author: vikranth
+---
+
+Open source is one of the most efficient ways in which one can contribute. I personally feel it to be a great way to contribute together with different people having different ideas.
+
+---
+
+## Climate change is one the most alarming things and if we dont take action now we may never get a chance to undo what we have done.
+
+![vikranth]({{site.baseurl}}/assets/images/shahil_akter.jpg)
```
Comment (tasnimzotder) [inline _hackbars/vikranth.md]: Another crawling :bug:

```suggestion
![vikranth]({{site.baseurl}}/assets/images/vikranth.jpg)
```

Comment (vikranthdatta) [reply _hackbars/vikranth.md]: I made all the changes you suggested.

**Review by tasnimzotder — APPROVED** [review]

File: _hackbars/vikranth.md (hunk 2 above)
Comment (tasnimzotder) [reply _hackbars/vikranth.md]: Thanks

**Comment by tasnimzotder** [comment]
Thanks @vikranthdatta, for your contributions :)

**Comment by vikranthdatta** [comment]
sorry for the trouble you had for editing my PR.

**Comment by tasnimzotder** [comment]
It's fine
We grow by other's growth...

**Pull request merged**